Python/bytecodes.c

```diff
@@ -1055,9 +1061,13 @@ dummy_func(
             gen->gi_exc_state.previous_item = NULL;
             _Py_LeaveRecursiveCallPy(tstate);
             _PyInterpreterFrame *gen_frame = frame;
+            gen_frame->instr_ptr = next_instr;
```

Comment:
Can you move this to the top of the instruction as `frame->instr_ptr = next_instr`.
That way the compiler can merge it with the `frame->instr_ptr = next_instr++;` in the `TARGET` macro.